SMLHelper/Utility/EnumCacheManager.cs

```diff
@@ -70,9 +72,44 @@ internal EnumCacheManager(string enumTypeName, int startingIndex, IEnumerable<in
                 }
 #endif
             }
+            else if (enumTypeName.CompareTo("CraftTreeType") == 0)
```

Comment:
Same review as Prime here, this stuff should be done in the respective patchers.